airflow/models/dag.py

```diff
@@ -685,6 +685,7 @@ def validate(self):
             )
         self.params.validate()
         self.timetable.validate()
+        self.validate_setup_teardown()
```

Comment:
Looks like we have some tests that have setup without teardown: https://github.com/apache/airflow/blob/5a4106d96e670b82004dd6fa221cf3c29b2496a9/tests/decorators/test_setup_teardown.py#L421-L423
Should we remove those because I think it gives the impression that they are supported?

Reply:
yeah we should.  i think ultimately we'll have to raise this on the list though.